PopPUNK/visualise.py

```diff
@@ -348,42 +351,6 @@ def generate_visualisations(query_db,
     # Generate MST
     mst_tree = None
     mst_graph = None
-    if tree == 'mst' or tree == 'both':
```

Comment:
I missed where this code went? Or has it been removed? (if so, why?)

Reply:
Not sure, restored and modified in 90144c0.